ui/app/components/mount-backend/type-form.hbs

```diff
@@ -29,5 +29,10 @@
 {{/each}}
 
 <div class="field is-grouped box is-fullwidth is-bottomless">
-  <Hds::Button @text="Cancel" @color="secondary" @route="vault.cluster.secrets.backends" />
```

Comment:
Small bug I noticed when testing, we were only going to the secret engine list on cancel